hu/testing.md

```diff
@@ -7,25 +7,25 @@
 <a name="introduction"></a>
 ## Bevezetés
 
-A Laravel a tesztelés szem előtt tartásával készült. Ez azt jelenti, hogy a PHPUnit támogatására fel van készítve az alkalmazásod és egy beállított `phpunit.xml` fájlt tartalmaz. A keretrendszer megfelelő segéd függvényekkel együtt került telepítésre, amelyekkel tesztelheted az alkalmazásodat.
+A Laravel a tesztelés szem előtt tartásával készült. Ez azt jelenti, hogy az alkalmazásod fel van készítve a PHPUnit támogatására, és tartalmaz egy előre beállított `phpunit.xml` fájlt. A keretrendszer megfelelő segéd függvényekkel együtt került telepítésre, amelyekkel tesztelheted az alkalmazásodat.
 
-Egy `ExampleTest.php` nevű fájl található a `tests` könyvtárban. Laravel alkalmazás installálása után, egyszerűen futtasd a tesztjeidet a `phpunit` paranccsal.
+Egy `ExampleTest.php` nevű fájl található a `tests` könyvtárban. A Laravel alkalmazás installálása után egyszerűen futtathatod a teszteket a `phpunit` paranccsal.
 
 <a name="environment"></a>
 ## Futtató környezet
 
-Tesztek futatásakor, a Laravel automatikusan a `testing` konfigurációs környezetet állítja be. A Laravel automatikusan beállítja a munkamenetet és gyorstárat az `array` meghajtóra a tesztek futása alatt, ami azt jelenti, hogy sem a munkamenetben sem a gyorstárban nem maradnak meg az adatok tesztfuttatás közben.
+Tesztek futatásakor a Laravel automatikusan a `testing` konfigurációs környezetet állítja be. Ilyenkor automatikusan beállítja a session-t és cache-t az `array` driverre, ami azt jelenti, hogy sem a munkamenetben, sem a gyorstárban nem maradnak meg az adatok teszt futtatása közben.
 
-Szabadon definiálhatsz más teszkörnyezet konfigurációs változókat amennyiben szükséges. A `testing` környezeti változókat a `phpunit.xml` fájlban állíthatod be, de figyelj arra, hogy a konfigurációs gyorstárad legyen kiürítve a `config:clear` Artisan paranccsal, mielőtt futtatod a tesztjeidet!
+Szabadon definiálhatsz más teszkörnyezet-konfigurációs változókat amennyiben szükséges. A `testing` környezeti változókat a `phpunit.xml` fájlban állíthatod be, de figyelj arra, hogy a konfigurációs cache legyen kiürítve a `config:clear` Artisan paranccsal a tesztek futtatása előtt!
 
 <a name="creating-and-running-tests"></a>
 ## Tesztek készítése és futtatása
 
-Új teszteset készítéséhez használd a `make:test` Artisan parancsot:
+Új tesztek készítéséhez használd a `make:test` Artisan parancsot:
```

Comment:
To create a new test case, use the make:test Artisan command:

nincs többes szám, és nem test, hanem test case. De tőlem maradhat.